lib/bibliotheca/schema/exercise.rb

```diff
@@ -31,7 +31,9 @@ def self.fields_schema
 
       {name: :description, kind: :file, extension: 'md'},
       {name: :hint, kind: :file, extension: 'md'},
-      {name: :corollary, kind: :file, extension: 'md'}
+      {name: :corollary, kind: :file, extension: 'md'},
+      {name: :initial_state, kind: :file, extension: 'md'},
+      {name: :final_state, kind: :file, extension: 'md'}
```

Comment:
@flbulgarelli Initial and final board should be `md` or `yml`?

Reply:
md

Reply:
Perfect!